contracts/mocks/tokens/InvalidReturnTokenMock.sol

```diff
@@ -41,13 +41,6 @@ contract InvalidReturnTokenMock {
     decimals = _decimals;
   }
 
-  /**
-  * @dev Total number of tokens in existence
-  */
-  function totalSupply() external view returns (uint256) {
```

Comment:
any reason this was removed?

Reply:
It is already defined as a public variable on line 15 here, so the getter should automatically be generated. it was giving me an error about shadowing variables here